notebooks/data-augmentation/wikidata-qa/wikidata.ipynb

```diff
@@ -0,0 +1,1208 @@
+      "\n",
+      "Q: And its use cases are?\r\n",
+      "A: ChatGPT is most commonly used for natural language generation and machine translation.\n",
+      "\n",
+      "Q: Who is reponsible for Generative Pre-trained Transformer?\r\n",
+      "A: OpenAI made it.\n",
+      "\n",
+      "Q: What license is associated with Generative Pre-trained Transformer?\r\n",
+      "A: Generative Pre-trained Transformer has the following license: proprietary license.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# chatgpt\n",
+    "print(wg.generate(\"Q115564437\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Questions about Bill Gates (American businessman and philanthropist (born 1955)):\n",
+      "\n",
+      "Q: How many children does Bill Gates have?\r\n",
+      "A: His 3 children are Jennifer Katherine Gates, Phoebe Adele Gates and Rory John Gates.\n",
+      "\n",
+      "Q: Who is his dad?\r\n",
+      "A: His name is Bill Gates Sr..\n",
+      "\n",
+      "Q: Who is his mother, respond with her name.\r\n",
+      "A: It's Mary Maxwell Gates.\n",
+      "\n",
+      "Q: Where is William Henry Gates III from?\r\n",
+      "A: He is from United States of America.\n",
+      "\n",
+      "Q: What does William Henry, III Gates work in?\r\n",
+      "A: William Henry, III Gates has multiple occupations: entrepreneur, programmer, computer scientist, inventor, financier, bridge player, investor, actor, philanthropist, writer and international forum participant.\n",
+      "\n",
+      "Q: What's the URL for William Henry \"Bill\" Gates III? Thanks!\r\n",
+      "A: His web address is: http://www.thegatesnotes.com/\n",
+      "\n",
+      "Q: his birthday is?\r\n",
+      "A: His birthday is on the 1955-10-28.\n",
+      "\n",
+      "Q: List his siblings please.\r\n",
+      "A: The number of brothers and sisters William Gates has is 2. Their names are Kristianne Gates and Libby Gates MacPhee.\n",
+      "\n",
+      "Q: Where was he born?\r\n",
+      "A: William H. Gates III was born in Seattle.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# bill gates\n",
+    "print(wg.generate(\"Q5284\", pronoun=\"he\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Questions and Answers on Budapest (also known as Buda Pest, Buda-Pest, Budapešť, Budapesta, Budapeszt, Buda, Ofen, Budín, Budim, Budon, Pest, Pešť, Pešta, Alt-Ofen, Budapest, Hungary):\n",
+      "\n",
+      "Q: What is Budapest's location?\r\n",
+      "A: The coordinates for Budapest are 47.498333333333 19.040833333333.\n",
+      "\n",
+      "Q: Do you know the date of its inception?\r\n",
+      "A: Budon was first released in 1873-11-17.\n",
```

Comment:
uh

Reply:
yep, you will often get some of these weird answers with this method, because there is a lot of ambiguity in the graph, that's why the replies need to be checked manually

Reply:
I know it's in the readme already but maybe a comment towards the top of the notebook noting this could be useful just in case people miss it in the readme